src/database/repo_service.py

```diff
@@ -17,6 +19,13 @@ def insert_new_code_repo(code_repo: RepoStorable):
                 github_db_object = Actions._map_github_repo_to_orm_object(code_repo)
                 if github_db_object:
                     session = db_manager.get_session()
+
+                    # TODO dont hardcode this to all languages
+                    languages = session.query(Language).all()
```

Comment:
As a suggestion for how to do this when the time comes....instead of hitting the DB and querying each language on the fly - I'd say hit the DB once - map the results to a dictionary keyed off of the language with a value of the db object and then as you go through the associated language apply as needed:

Ex:
```
languages = ls_action.get_all_languages()
languages_mapping_dictionary =  {key: language.name.lower() for language in languages}
```

Then as you create or update your repos you can do this
```
for repo in repos:
     languages_to_relate = some_mapper_for_languages[repo.id]
     for related_language in languages_to_relate:
         repo.languages.append(languages_mapping_dictionary[language_name])
```
1 SQL query gets you all that you need.